geth/mailservice/mailservice_api.go

```diff
@@ -16,6 +17,14 @@ import (
 const (
 	// defaultWorkTime is a work time reported in messages sent to MailServer nodes.
 	defaultWorkTime = 5
+
+	// gcTimeout defines timeout to release not used connection with meailserver peers.
+	gcTimeout = 5 * time.Minute
```

Comment:
I'm wondering how that would work with the pausing/resuming activity on iOS/Android.

Reply:
should work fine, cause even if gc will disconnect a peer - we will enforce connection with it on next historic messages request

btw it is not long-term solution, this is just how things are handled now on react side. i want to preserve current behavior to solve the problem described in PR and then move to a solution that will allow every peer to have at least one persistent connection with mail server